themes/README.rst

```diff
@@ -154,3 +176,56 @@ name in the ``@import`` line.
 Run the ``update_assets`` command to recompile the theme::
 
     $ paver update_assets lms --settings=aws
+
+Microsites
+==========
+
+If you want to continue using the "Microsites" theming system, there are a few
+changes you'll need to make. A few templates have been renamed, or folded into
+other templates:
+
+* ``header_extra.html`` has been renamed to ``head-extra.html``. This file
+  was always inserted into the ``<head>`` element of the page, rather than
+  the header of the ``<body>`` element, so this change makes the name more
+  accurate.
+
+* ``google_analytics.html`` has been removed. The contents of this template
```

Comment:
Why was this file removed? I like the convention of having one file per vendor, as it makes it easy to find and work with. Was there a constraint that made this necessary? I don't like the idea of having one monolithic template to understand.